tests/backup/backup_test.go

```diff
@@ -6692,3 +6693,155 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
 		DeleteCloudAccounts(backupLocationMap, cloudAccountName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{CustomResourceRestore}", func() {
+	var (
+		contexts           []*scheduler.Context
+		appContexts        []*scheduler.Context
+		backupLocationUID  string
+		cloudCredUID       string
+		bkpNamespaces      []string
+		clusterUid         string
+		clusterStatus      api.ClusterInfo_StatusInfo_Status
+		backupName         string
+		credName           string
+		cloudCredUidList   []string
+		backupLocationName string
+		deploymentName     string
+		restoreName        string
+		backupNames        []string
+		restoreNames       []string
+	)
+	labelSelectors := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	newBackupLocationMap := make(map[string]string)
+	backupNamespaceMap := make(map[string]string)
+	deploymentBackupMap := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("CustomResourceRestore", "Create custom resource restore", nil, 58041)
+		log.InfoD("Deploy applications")
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Create custom resource restore", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			log.InfoD("Validate applications")
+			ValidateApplications(contexts)
+		})
+
+		Step("Creating credentials and backup location", func() {
+			log.Infof("Creating cloud credentials and backup location")
+			for _, provider := range providers {
+				cloudCredUID = uuid.New()
+				cloudCredUidList = append(cloudCredUidList, cloudCredUID)
+				backupLocationUID = uuid.New()
+				credName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+				CreateCloudCredential(provider, credName, cloudCredUID, orgID)
+				log.InfoD("Created Cloud Credentials with name - %s", credName)
+				backupLocationName = fmt.Sprintf("autogenerated-backup-location-%v", time.Now().Unix())
+				newBackupLocationMap[backupLocationUID] = backupLocationName
+				err := CreateBackupLocation(provider, backupLocationName, backupLocationUID, credName, cloudCredUID, getGlobalBucketName(provider), orgID, "")
+				log.FailOnError(err, "Creating Backup location %v", backupLocationName)
```

Comment:
since this will be logged only when there is error. make description line. "creating backup location [%v] failed"

please have a look at other similar error messages